app/models/ecf_participant_eligibility.rb

```diff
@@ -6,6 +6,8 @@ class ECFParticipantEligibility < ApplicationRecord
   belongs_to :participant_profile, class_name: "ParticipantProfile::ECF", touch: true
   before_validation :determine_status, on: :create
 
+  scope :updated_after, ->(timestamp) { where(updated_at: ..timestamp) }
```

Comment:
Isn't this `updated_before` otherwise it'd be `where(updated_at: timestamp..)` ?

Reply:
Yes it is updated_before, thats the correct logic as well, not sure why my brain decided to call it updated_after...